package.json

```diff
@@ -29,11 +29,11 @@
     "eslint": "^7.32.0"
   },
   "peerDependencies": {
-    "@reduxjs/toolkit": "^1.9.5",
+    "@reduxjs/toolkit": "^2.2.1",
     "react": ">= 16",
-    "react-redux": "^7.2.8 || ^8.0.0",
+    "react-redux": "^7.2.8 || ^8.0.0 || ^9.1.0",
```

Comment:
It's really weird that `react-redux@7` doesn't specify redux as a peer dependency... That being said, we know that v8 supports redux 4 and 5, so I think we should drop support for `react-redux@7` to be safe.